backuprestore.adoc

```diff
@@ -1192,7 +1193,7 @@ rules.
 {product-name}.
 .  Copy the backup files to the new server.
 .  Run `zmrestoreldap` to restore the global LDAP data.
-.  Run `zmrestoreoffline` to restore account data from the backup
+.  Run `zmrestoreoffline` to restore account data from the backed up
```

Comment:
nit: for consistency keep "backup session" instead of "backed up sessions"
bonus points: change a the remaining five (5) "back up" occurrences in the doc to "backup"
Example:
> Perform a manual back up all mailboxes...

becomes: Perform a manual backup of all mailboxes...

Another example:

> Back up using the Auto-Grouped Method

becomes: Backup using the Auto-Grouped Method